akita/__tests__/entity-store.spec.ts

```diff
@@ -62,17 +62,15 @@ describe('EntitiesStore', () => {
       store.add(new Todo({ id: 2 }));
       store.add(new Todo({ id: 3 }));
       store.add(new Todo({ id: 0 }), { prepend:true });
-      expect(store.entities[0].id).toEqual(0);
+      expect(Object.keys(store.entities)).toEqual([0, 1, 2, 3]);
```

Comment:
check store.ids not `Object.keys(store.entities)`

Reply:
![image](https://user-images.githubusercontent.com/16970990/47779546-e891c680-dd2b-11e8-94a5-1dc19ff4f3f3.png)
I dont see .ids. is that exist?

Reply:
store._value().ids